src/osd/ReplicatedBackend.cc

```diff
@@ -1363,8 +1365,9 @@ void ReplicatedBackend::prepare_pull(
   assert(!q->second.empty());
 
   // pick a pullee
+  // boost::random_device rng;
+  // boost::mt19937 urng(rng());
```

Comment:
@wjwithagen you removed the random_shuffle. based on your change, i am posting #19872. could you help take a look?